### DISCUSSION TITLE
Non-IT Port types

### DISCUSSION THREAD

**Discussion by Ivarrrrr** [netbox-community/devicetype-library#1686] [Q&A] [answered]
I wish to add a few port types for Video- and Audio-connectivity. I see the BNC-connector is available, but i wish to add Jack-ports (jack-tr, jack-trs, jack-trrs) and micro-bnc. 
Do i need to add these terminal-types myself? Or will they be auto-added if i use/describe them in a new device-type?

**Comment by DanSheps** [comment, marked as answer]
> I wish to add a few port types for Video- and Audio-connectivity. I see the BNC-connector is available, but i wish to add Jack-ports (jack-tr, jack-trs, jack-trrs) and micro-bnc.

Video/Audio is outside of NetBox's scope

> Do i need to add these terminal-types myself? Or will they be auto-added if i use/describe them in a new device-type?

You can typically customize field choices in NetBox ([Documentation](https://docs.netbox.dev/en/stable/configuration/data-validation/#field_choices)) however interface types are not customizeable and must be submitted for inclusion through the netbox feature request process.

> Do i need to add these terminal-types myself? Or will they be auto-added if i use/describe them in a new device-type?

We will not entertain any PR's with non-netbox standard interface types.  I would suggest using "other"
